app/config/index.js

```diff
@@ -191,7 +191,7 @@ const config = convict({
   CHECK_TAGS: {
     doc: 'Enable checking tags on s3 object',
     format: Boolean,
-    default: false,
+    default: true,
```

Comment:
I assume this is ok now that permissions have been added for object tags?

Reply:
Yes